lib/template-mixins.js

```diff
@@ -528,6 +530,15 @@ module.exports = function (options, deprecated) {
             };
         };
 
+        res.locals.qs = function () {
+            return function (query) {
+                query = querystring.stringify(Object.assign({}, req.query, querystring.parse(query)));
+                var urlObj = url.parse(req.url);
+                urlObj.search = '?' + query;
+                return url.format(urlObj);
```

Comment:
Is it necessary to prefix with the full url? It feels like the function would be *more* useful if it only returned the querystring component prefixed with `?` since that can be used just as well for relative links to the same url, but could also be used for linking to other base urls.